UI.Dockerfile

```diff
@@ -0,0 +1,52 @@
+# SPDX-FileCopyrightText: 2024 HH Partners
+# 
+# SPDX-License-Identifier: MIT
+
+FROM node:20.10.0 AS base
+
+FROM base AS deps
+WORKDIR /app/dos
+
+COPY package.json ./package.json
+COPY apps/clearance_ui/package.json ./apps/clearance_ui/package.json
+COPY packages/common-helpers/package.json ./packages/common-helpers/package.json
+COPY packages/s3-helpers/package.json ./packages/s3-helpers/package.json
+COPY package-lock.json ./package-lock.json
+
```

Comment:
Here's the package.json files that the UI will need:
`COPY package.json ./package.json
COPY apps/clearance_ui/package.json ./apps/clearance_ui/package.json
COPY packages/spdx-validation/package.json ./packages/spdx-validation/package.json
COPY packages/validation-helpers/package.json ./packages/validation-helpers/package.json
COPY package-lock.json ./package-lock.json`